### PR TITLE
Add more music icons and another mic icon

### PR DESCRIPTION
**Pull request by it-is-not**
music and mic icon

### CODE REVIEW

**Review by ericfennis — APPROVED** [review]
@it-is-not Nice icons!

File: icons/music-2.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M8 22a4 4 0 1 0 0-8 4 4 0 0 0 0 8Z" />
```
Comment (karsa-mistmere) [inline icons/music-2.svg:12]: I'd say let's change these to `circle` tags?

File: icons/mic-2.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 8-9.17 9.17a2.83 2.83 0 1 0 4 4L16 12" />
```
Comment (karsa-mistmere) [inline icons/mic-2.svg:12]: There's an ever so tiny design guideline violation regarding the 1px padding. I could't get things to work with the SVG optimization on, it kept messing up the lines, but I got this to work, what do you think?
```suggestion
  <path d="M12.02 8 2.65 18.23a2.2 2.2 0 0 0 3.12 3.12L16 11.98" />
```

Comment (karsa-mistmere) [reply icons/mic-2.svg:12]:
```suggestion
  <path d="m12 8-9.04 9.06a2.82 2.82 0 1 0 3.98 3.98L16 12" />
```
Oooor, just never mind, I think I got it to work with the original design. :)

Comment (it-is-not) [reply icons/mic-2.svg:12]: Good point, I like the shape of the new one but I think it should be a bit more like the `edit` or `brush` icons for consistency so I made something that's in-between that also follows the rules. Should I update the icon with this one or your one?

image

Comment (karsa-mistmere) [reply icons/mic-2.svg:12]: I think this should be pretty much the same as my second comment?
```suggestion
  <path d="m12 8-9.04 9.06a2.82 2.82 0 1 0 3.98 3.98L16 12" />
```

image

**Comment by karsa-mistmere** [comment]
@it-is-not: Hey, I've just noticed that you opened this PR to merge into `master`, but we've switched to `main` recently, could you open a new PR so that I can merge that and you still get properly credited?

**Pull request merged**